docs/extensions/FrameworkElementEx.md

```diff
@@ -0,0 +1,42 @@
+---
+title: FrameworkElement extentions
+author: ST-Apps
+ms.date: 01/05/2018
+description: FrameworkElementEx provides a simple way to bind to actual size for any FrameworkElement
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, FrameworkElement, extentions
+---
+
+# FrameworkElement extentions
+
+FrameworkElementEx provides a simple way to bind to actual size for any FrameworkElement
+
+## EnableActualSizeBinding
+
+The EnableActualSizeBinding property allows to enable/disable the binding for ActualHeight and ActualWidth
+
+## ActualHeight
+
+The ActualHeight property allows to bind to TargetObject's ActualHeight
+
+### Example
+
+```xaml
+    <Rectangle x:Name="TargetObject"
+               extensions:FrameworkElementEx.EnableActualSizeBinding="true"/>
+	...
+	<TextBlock Text="{Binding ElementName=TargetObject, Path=(extensions:FrameworkElementEx.ActualHeight)}" />
+```
+
+## ActualWidth
+
+The ActualWidth works exactly as ActualHeight.
+
+## Requirements (Windows 10 Device Family)
+
+| [Device family](http://go.microsoft.com/fwlink/p/?LinkID=526370) | Universal, 10.0.14393.0 or higher |
```

Comment:
(aside) @nmetulev should we have Toolkit version introduced in the docs in this section too?  I know it may be a bit of a project to investigate, but I know with the docs.microsoft.com now they're starting to call out when specific properties are added with each version.  It could be useful for us to have as we add more.  Thoughts?